lib/Controller/FaceController.php

```diff
@@ -57,6 +57,7 @@ public function getThumb ($id, $size) {
 		$img = new OCP_Image();
 		$fileName = $ownerView->getLocalFile($path);
 		$img->loadFromFile($fileName);
+		$img->fixOrientation();
```

Comment:
sweet:)

Reply:
D'Oh!. 
I mixed that change that i was testing.. 😅
If confirm that fix nc15, I merge it like this.😌